docs/interaction-spreadsheet.md

```diff
@@ -1,52 +1,54 @@
 ## Spreadsheet interaction model structure
+
 You can take a look at `example-interaction-model.xlsx` in the root of the repository
-* Spreadsheet must contain a valid local on it's name eg. `MySkill - Intents & Utterances-en-US`. Valid Locales are (['en-US','en-GB', 'de-DE'])
-* Tab for intent must be named `INTENT`
-* Tab for utterances must be named `UTTERANCES` eg. `UTTERANCES_MAIN`, `UTTERANCES_HELP`
-* Tab for slots must contain `LIST_OF_` eg. `LIST_OF_TERMS`.
-* If your slots contains synonym add a column named synonym
-* Tab for invocation name must be named `INVOCATION_NAMES`
+
+- Spreadsheet must contain a valid local on it's name eg. `MySkill - Intents & Utterances-en-US`. Valid Locales are (['en-US','en-GB', 'de-DE'])
+- Tab for intent must be named `INTENT`
+- Tab for utterances must be named `UTTERANCES` eg. `UTTERANCES_MAIN`, `UTTERANCES_HELP`
+- Tab for slots must contain `LIST_OF_` eg. `LIST_OF_TERMS`.
+- If your slots contains synonym add a column named synonym
+- Tab for invocation name must be named `INVOCATION_NAMES`
+- Tab for downloads name must be named `DOWNLOAD_`
 
 ### slots must have the following Structure
 
-LIST_OF_TERMS | synonym
---- | ---
-rain | rain
-rainy day | rain
-rainstorm | rain
-rainfall | rain
+| LIST_OF_TERMS | synonym |
```

Comment:
this is an usual source of confusion, most people don't get that the this column actually means "synonym of"

Reply:
Agreed. I'll create an issue and address it in another PR

Reply:
#43 ^^